apps/public-reference/pages/applications/view.tsx

```diff
@@ -18,13 +18,20 @@ export default () => {
     return moment().format(DATE_FORMAT)
   }, [])
 
+  const [origin] = useState<string>(typeof window !== "undefined" ? window.location.origin : "")
```

Comment:
Is there a reason to wrap this in `useState`?

Reply:
nope, it can be just a value, without useState, refactored.